src/qililab/platform/platform.py

```diff
@@ -637,14 +643,11 @@ def execute(
                 instrument.desync_sequencers()
 
         # FIXME: set multiple readout buses
-        if len(results) > 1:
-            logger.error("Only One Readout Bus allowed. Reading only from the first one.")
-        if not results:
-            raise ValueError("There are no readout buses in the platform.")
-
```

Comment:
Why are you removing these lines?

Reply:
It was easier to implement the logic in the compiler assuming only one qrm is present, which can be changed to multiple qrms but would be out of the scope of this pr. However since I am adding this one qrm restriction in the compiler:
```python
        if (
            sum(qblox_module.name in self.readout_modules for qblox_module in self.qblox_modules) > 1
        ):  # FIXME: only one qrm supported
            raise NotImplementedError(
                f"Only one readout module can be connected, found {sum(qblox_module.name in self.readout_modules for qblox_module in self.qblox_modules)} instead"
            )
```
we dont need to raise an error here, since it will be raised before (at the compiler)

Reply:
Note that there is also a separate error being raised at the very beginning of compilation if no QRM is found:
```python
        if all(
            qblox.name not in self.readout_modules for qblox in self.qblox_modules
        ):  # Raise error if qrm is not found
            raise ValueError(
                f"At least one QRM should be connected. Found {sum(qblox.name in self.readout_modules for qblox in self.qblox_modules)} QRM modules"
            )
```